dotcom-rendering/src/components/marketing/hooks/useTicker.ts

```diff
@@ -1,12 +1,7 @@
-/**
```

Comment:
can this file be removed?

Reply:
yes good spot it's not being used anymore